packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -132,7 +132,7 @@ export enum ReducerID {
 }
 
 export function getFieldTypeForReducer(id: ReducerID, fallback: FieldType): FieldType {
```

Comment:
@gelicia would you mind adding a few tests for the special types this handles?

Reply:
I plan to do this today but as a follow up so this can get in faster.